contracts/protocol/libraries/position/PositionHouseFunction.sol

```diff
@@ -786,7 +788,7 @@ library PositionHouseFunction {
         );
         positionResp.realizedPnl =
             (unrealizedPnl * int256(positionResp.exchangedPositionSize)) /
```

Comment:
realized pnl of reverse position calculated by unrealized pnl * | reduce quantity / total quantity |. but old code using total quantity in int256, what can be negative if position side is short. fixed by using the absolute value of the total quantity